src/environmentd/src/http/webhook.rs

```diff
@@ -37,6 +35,10 @@ pub async fn handle_webhook(
     headers: http::HeaderMap,
     body: Bytes,
 ) -> impl IntoResponse {
+    // Record the time we receive the request, for use if validation checks the current timestamp.
+    let received_at = client.now();
+    let conn_id = client.new_conn_id().context("allocate connection id")?;
+
```

Comment:
@matthewarthur I suspect all the changes to this file are spurious—a result of a rebase gone wrong, perhaps? Things should work if you back out the changes to this file.